Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -2462,6 +2462,7 @@ SELECT albumValues.ItemId
                 if (query.SearchTerm.Length > 1)
                 {
                     builder.Append("+ ((CleanName like @SearchTermContains or (OriginalTitle not null and OriginalTitle like @SearchTermContains)) * 10)");
+                    builder.Append("+ ((Tags not null and Tags like @SearchTermContains) * 5)");
```

Comment:
Not absolutely necessary, but you could do this:
```suggestion
                    builder.
                        Append("+ ((CleanName like @SearchTermContains or (OriginalTitle not null and OriginalTitle like @SearchTermContains)) * 10)").
                        Append("+ ((Tags not null and Tags like @SearchTermContains) * 5)");
```